checker.go

```diff
@@ -6,6 +6,14 @@ import (
 	"strings"
 )
 
+type sign int
```

Comment:
This package is already v1 - meaning we cannot just get rid of `Positivity`, `Negativity` and `Analyze`. We can expand it however. I guess the safe move here is extends this package interface with a single scanner that provides all the feelings and data, and wrap it with these specific calls. Essentially reading the calculated sentiment. Right now, Analyze is scanning the text twice - this will be the biggest gain.